cfgov/v1/wagtail_hooks.py

```diff
@@ -110,3 +113,57 @@ def register_django_admin_menu_item():
         classnames='icon icon-redirect',
         order=99999
     )
+
+
+class RelativePageLinkHandler(PageLinkHandler):
+    """
+    Rich text link handler that forces all page links to be relative.
+
+    This special page link handler makes it so that any internal Wagtail page
+    links inserted into rich text fields, for example:
+
+        <a linktype="page" id="123">foo</a>
+
+    always get replaced with a relative URL to that page, never an absolute
+    one, like this:
+
+        <a href="/path/to/page">foo</a>
```

Comment:
The above explanation could maybe be a little clearer about the fact that the first example is the database representation, and the second example is the final rendering to the client.

Also, the phrasing makes it unclear to me if the second example is a relative URL or an absolute URL. Consider including an example of each?